src/OpenRiaServices.DomainServices.Server/Framework/Data/DomainService.cs

```diff
@@ -303,11 +303,9 @@ public async virtual ValueTask<ServiceQueryResult<T>> QueryAsync<T>(QueryDescrip
                 {
                     try
                     {
-                        result = queryDescription.Method.Invoke(this, parameters, out totalCount);
-                        if (queryDescription.Method.IsTaskAsync)
-                        {
-                            result = await queryDescription.Method.UnwrapTaskResult(result).ConfigureAwait(false);
-                        }
+                        result = await queryDescription.Method
```

Comment:
nit: You changed the style below with the call on the first line.
Did you intentionally leave his as is because it was to long otherwise ?